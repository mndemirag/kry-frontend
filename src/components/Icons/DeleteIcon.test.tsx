import { render } from "@testing-library/react";

import { DeleteIcon } from "./DeleteIcon";

describe("DeleteIcon", () => {
  it("renders properly", () => {
    const { container } = render(<DeleteIcon />);
    expect(container).toMatchSnapshot();
  });
});
