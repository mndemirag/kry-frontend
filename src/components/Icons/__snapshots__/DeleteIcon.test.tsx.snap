// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteIcon renders properly 1`] = `
<div>
  <div
    id="deleteIcon"
  >
    <svg
      aria-hidden="true"
      height="24px"
      viewBox="0 0 48 48"
      width="24px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <polyline
        fill="#f55376"
        points="12.229,42.048 5.952,35.77 35.771,5.952 42.048,12.23 12.229,42.048"
      />
      <path
        d="M30.278,24L24,30.278l11.771,11.77l6.277-6.278L30.278,24 M12.229,5.952L5.952,12.23L17.722,24 L24,17.722L12.229,5.952"
        fill="#f55376"
      />
      <polyline
        fill="#eb0000"
        points="24,17.722 17.722,24 24,30.278 30.278,24 24,17.722"
      />
    </svg>
    <span
      class="visuallyHidden"
    >
      Delete the service
    </span>
  </div>
</div>
`;
